tools/test-backend

```diff
@@ -140,6 +140,7 @@ not_yet_fully_covered = {path for target in [
 enforce_fully_covered = sorted(target_fully_covered - not_yet_fully_covered)
 
 FAILED_TEST_PATH = 'var/last_test_failure.json'
+FAILED_THREAD_PATH = 'var/failed_thread_tests.json'
 
 def get_failed_tests() -> List[str]:
```

Comment:
I was thinking this could be changed to `Optional[List[str]]` as well.  It seems a more appropriate typing considering the empty list is just used to check for falsey-ness when deciding to populate `args`.